Translation/en/RedirectedResources/assets/abdata/adv/scenario/c05/00/60/translation.txt

```diff
@@ -1,10 +1,10 @@
 //
 // Dumped for Koikatsu Sunshine v1.0.2 (KoikatsuSunshine) by KKS_TextDump v1.4.5.2
 //
-//「ねぇ……部屋にいる？」=
-//SELECTION:「返事をする」=
-//SELECTION:「……」=
-//「どうかしたの？」=
-//「“かれら”が……ううん、わたしの意思で、あなたに会いに来たの」=
-//「今夜はずっと……あなたの傍にいたい……」=
-//「はぁ……もう寝たのかな」=
+「ねぇ……部屋にいる？」=“Hey... are you in there?”
+SELECTION:「返事をする」=SELECTION: “Answer her”
+SELECTION:「……」=SELECTION: “..."”
```

Comment:
```suggestion
SELECTION:「返事をする」=“Answer her”
SELECTION:「……」=“...”
```

Reply:
Special prefixes like SELECTION: or OPTION: shouldn't be put in the translation.